include/tester/db/VTDatabase.hpp

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include "Database.hpp"
+
+class VTDatabase : Database
+{
+public:
+    VTDatabase(const std::string &domain):
+        Database(domain)
+    {     
+    }
+
+    ~VTDatabase() = default;
+
+    void save(const bsoncxx::document::value &document)
+    {
+        Database::save(COLLECTION_NAME ,document);
+    }
+
+    bsoncxx::document::value getAllHeaders()
```

Comment:
I'd suggest giving this a better name, since this is relevant to the `history` feature you added - how about a simple `getHistory`? or better yet - `getTestLogsHistory`?

Reply:
Done